### ISSUE TITLE
Any way to stop orgid "is now known as" messages?

### ISSUE THREAD

**Issue by robghen** [EionRobb/purple-teams#36]
The plugin works for me but I get a LOT of messages like this in all IM tabs:

    orgid:de69e5b6-3b33-3801-9733-3f088104e728 is now known as Doe, Jane.

These come pretty randomly but at a high enough volume that the real messages get lost among all these orgid messages.

I tried using the pidgin-ignore-nickchange plugin but didn't have any luck getting that to work so wanted to see if anyone has any ideas where these things are coming from and how to make them stop.  I had previously been using the Skype/sip plugin and never saw these until I changed to using Teams instead.

Thanks for any help!

**Comment by EionRobb** [comment]
Looks to be solved in this commit ^ :tada:

**Comment by robghen** [comment]
I just installed the latest version and that resolved the issue.  Interestingly, there was one user where it was still streaming these messages at a high rate.  I happen to notice it was showing something like:

(04:56:52 PM) orgid:de69f5b6-3b33-4701-9733-2e088104e728 is now known as Doe, Jane.
(04:56:50 PM) Doe, Jane A: Some message here

It caught my eye that the alias for that buddy had the middle initial and the 'now known as' did not.  I removed the middle initial from the alias for that account and that resolved the issue for that one as well.  I just wanted to note that in case someone else stumbles on this and is having a similar problem.

Thank you SO much for the fix, I'm all set!
